### DISCUSSION TITLE
Some submarines spawn (way) outside the map when mirrored for pvp

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15046] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

[SvS beacon vs beacon.zip](https://github.com/user-attachments/files/17609020/SvS.beacon.vs.beacon.zip)

for example, attempting to use this submarine while team2 (seperatist) in either sub v sub or KoTH mode will result in them spawning way in incorrect spots. 

![image](https://github.com/user-attachments/assets/f76d63dc-2f9a-42de-8b9c-a24357835445)
![image](https://github.com/user-attachments/assets/46edc6f5-326f-412e-a5d2-353e02f2a980)
![image](https://github.com/user-attachments/assets/3f4e3b10-aedb-4372-9e52-beb01d778086)

as you can see, blue team spawns correctly.

### Reproduction steps

1. use that sub
2. join seperatist team in sub v sub
3. go in freecam and debugai, see how weirdly it spawns

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by AziKarnage** [comment]
If this gets fixed I will give the devs one whole dollar

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.

**Reply by Regalis11** [reply]
The fix unfortunately didn't make it in time for the next update, but will be included in a future one.
